presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+
+* Fix a correctness bug for queries with a ``LIMIT`` on top of a ``FULL OUTER JOIN``.
+* Fix a correctness bug in how word boundaries are handled for Joni regular expression functions.
+* Fix a correctness bug that would occur if two tables with mismatching but compatible bucket counts are joined whiles filters are applied to one of the tables.
+* Fix Parquet predicate pushdown for ``smallint`` and ``tinyint`` types.
+* Fix a failure that occurs when view columns were fully qualified with catalog and/or schema.
+* Fix a failure that occurs then creating and reading an unpartitioned Hive table within the same transaction.
+* Fix an issue that causes procedure calls to fail if no queries have been run after the server startup.
+* Fix an issue in analyzing property system table. If a connector is removed, its analyze properties will also be removed in the ``system.metadata.analyze_properties`` table.
```

Comment:
I don't understand this.  Would the following be a correct description?:
```
Fix an issue where analyze properties would remain in the ``system.metadata.analyze_properties`` table even after the connector was removed.
```

Reply:
@jessesleeping